app/mailers/requests_confirmation_mailer.rb

```diff
@@ -24,7 +24,7 @@ def combined_items(request)
     grouped = request.request_items.group_by { |i| i['item_id'] }
     # convert hash into an array of items with combined quantities
     grouped.map do |id, items|
-      { 'item_id' => id, 'quantity' => items.map { |i| i['quantity'] }.sum }
+      { 'item_id' => id, 'quantity' => items.map { |i| i['quantity'] }.sum, name: Item.find(id).name }
```

Comment:
Missed this... we're still doing a separate query per item this way. We're basically overwriting this name with the one from the single query above :) I think this change probably should be reverted.

Reply:
Right... this was a remnant from my initial solution.  Sorry.  Rushed too much.